content/scalability/docs/cluster-services.md

```diff
@@ -0,0 +1,60 @@
+# Cluster Services
+
+Cluster services run inside an EKS cluster, but they are not user workloads. If you have a Linux server you often need to run services like NTP, syslog, and a container runtime to support your workloads. Cluster services are similar, supporting services that help you automate and operate your cluster. In Kubernetes these are usually run in the kube-system namespace and some are run as DaemonSets.
+
+Cluster services are expected to have a high up-time and are often critical during outages and for troubleshooting. They should run on dedicated compute instances such as a separate node group or AWS Fargate. This will ensure that the cluster services are not impacted on shared instances by workloads that may be scaling up or using more resources.
+
+## Scale CoreDNS
+
+Scaling CoreDNS has two primary mechanisms. Reducing the number of calls to the CoreDNS service and increasing the number of replicas.
+
+### Reduce external queries by lowering ndots
+
+The ndots setting specifies how many periods in a domain name are considered enough to avoid querying DNS. If your application has an ndots setting of 5 (default) and you request resources from api.example.com (http://api.example.com/) (2 dots) then CoreDNS will be queried for each search domain defined in /etc/resolv.conf for a more specific domain (e.g. api.example.com.namespace.svc.) until all searches are exhausted or a match is found.
```

Comment:
I assume that api.example.com is an external service (outside the cluster). I don't think it's a best practices to use internet domain names as part of your namespace. 
It's a solid recommendation. Can you explicitly say how many DNS queries will be issued when ndots is 2 and ndots is 5, perhaps showing which names will queried?